src/main/resources/db.scripts.mvp/update_visit_times_mhi.sql

```diff
@@ -0,0 +1,43 @@
+-- This sql makes sure visits in future aline up to the latest session times and session templates.
+
+UPDATE visit
```

Comment:
do we need to do this? the ticket says  - Make this live effectively immediately - any booked visits will only have a minor impact